common/main/en.xml

```diff
@@ -5795,15 +5795,15 @@ annotations.
 				<displayName count="other">Slovak korunas</displayName>
 			</currency>
 			<currency type="SLE">
-				<displayName>Sierra Leonean New Leone</displayName>
-				<displayName count="one">Sierra Leonean new leone</displayName>
-				<displayName count="other">Sierra Leonean new leones</displayName>
-			</currency>
-			<currency type="SLL">
 				<displayName>Sierra Leonean Leone</displayName>
 				<displayName count="one">Sierra Leonean leone</displayName>
 				<displayName count="other">Sierra Leonean leones</displayName>
 			</currency>
+			<currency type="SLL">
+				<displayName>Sierra Leonean Leone (1964-2022)</displayName>
+				<displayName count="one">Sierra Leonean leone (1964-2022)</displayName>
+				<displayName count="other">Sierra Leonean leones (1964-2022)</displayName>
+			</currency>
```

Comment:
Good change for display name...

Reply:
But the test seems to be complaining that these are not en-dashes....

Reply:
OK, yeah trying to work on this and bits in between meetings… I'll try again

Reply:
And TestSupplementalInfo.TestCurrency seems to be complaining that the name has a year range even though it is still in current use as of today< that might need a test exception.

Reply:
It could check to see if this year is the final year.

Reply:
Oh, good idea

Reply:
@pedberg-icu Hah…  changing hyphen-minus to em-dash incorrectly quells the "Error: Has 'old' in name" because it didn't check for em-dash.

Reply:
I am happy to just go with that and have a separate ticket to fix the test to check for en-dash